### DISCUSSION TITLE
OSPF default interface passive

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1573] [Features]
Cisco provides a way to [configure OSPF interfaces as passive by default]( https://www.cisco.com/c/en/us/td/docs/routers/ios/config/17-x/ip-routing/b-ip-routing/m_iri-default-passive-interface.html#GUID-AE734874-728B-48D8-AC8D-14C74910DB5A ), which is useful for configurations with many VLAN interfaces

I propose that Netlab support the same, by allowing users to configure ```ospf.passive: True``` as a global default

**Comment by ipspace** [comment]
We could add 'passive' as global/node attribute that is copied into interfaces (same as OSPF area), but what's wrong with VLAN groups?

**Reply by jbemmel** [reply]
Nothing, but [preaching to the choir] OSPF is independent of VLANs and there could be a mix of interfaces, all of which I'd want to be passive by default

**Reply by ipspace** [reply]
OK. Submit a PR.

**Comment by jbemmel** [comment]
Implemented in https://github.com/ipspace/netlab/pull/1612
